compiler/cgen.nim

```diff
@@ -1362,6 +1362,7 @@ proc genInitCode(m: BModule) =
       add(prc, genSectionEnd(cpsStmts, m.config))
     addf(prc, "}$N", [])
 
+  addf(prc, "{$N", [])
```

Comment:
Minor nit, can you move lines 1330-1338 below this `addf`? This way all the `initProc` codegen is grouped together.

Reply:
ok, the lines are moved.